privacy-policy.txt

```diff
@@ -0,0 +1,55 @@
+[B]Kodi Privacy Policy[/B]
+[I]2016 Oct 23[/I]
+
+Your privacy is an important factor that the XBMC Foundation and Team Kodi considers in the development of all of our software and services. We are committed to being transparent and open. This Privacy Policy explains generally how we receive information about you, and what we do with that information once we have it.
+
+
+[B]What do we mean by “personal information?”[/B]
+
+For us, “personal information” means information which identifies you, like your name or email address.
+
+Any information that falls outside of this is “non-personal information.”
+
+If we store your personal information with information that is non-personal, we will consider the combination as personal information. If we remove all personal information from a set of data then the remaining is non-personal information.
+
+
+[B]How do we learn information about you?[/B]
+
+We learn information about you when:
+
+    • you give it to us directly (e.g., when you choose to send us logs in the forums);
+    • we collect it automatically through our software and services (e.g., when your Kodi connects with our servers to update add-ons, or in download statistics provided by Google Play);
+    • when we try and understand more about you based on information you’ve given to us (e.g., when we use your platform information provided to the add-on server to figure out which platforms use Kodi the most, to better understand where we should focus our efforts).
+
+
+[B]What do we do with your information once we have it?[/B]
+
+Generally, we use your information to help us provide and improve our software and services for you (e.g., we use a log you send us to figure out why Kodi isn’t playing a video right or why it might have crashed, or we determine how many active users are using each platform in order to determine how to allocate resources per platform).
+
+
+[B]When do we share your information with others?[/B]
+
+    • When we have asked and received your permission to share it.
+    • When we are fulfilling our educational purpose. We sometimes publicly release information to make our software better and foster an open web, but when we do so, we will remove your personal information and try to disclose it in a way that minimizes the risk of you being re-identified. For example, here is a blog post where we attempt to analyze from available data how many active users of XBMC existed at the time and share that info with the community.
```

Comment:
I suggest we change:

`For example, here is a blog post where we attempt to analyze...`

to:

`For example, in a blog post we might attempt to analyze...`

Also, XBMC -> Kodi